views/frontend/checkout-core/src/utils/hideSeparateFields.ts

```diff
@@ -17,6 +17,8 @@ export const hideSeparateFields = (): void => {
    * Classic checkout must be handled here.
    */
   if (isClassicCheckout()) {
+    // Hide separate address fields as a precaution when they shouldn't be visible
```

Comment:
Ik denk dat de functie wel afdoende is uitgelegd in de comment die er al staat toch (doc block hierboven)? Als jij dat ook vindt zou ik deze en de volgende regel gewoon weghalen. Als het niet duidelijk is, graag de comments proberen te combineren in iets wat wel duidelijk is.